.buildkite/benchmark.yml

```
@@ -19,12 +20,24 @@ steps:
       GROUP: "CUDA"
     timeout_in_minutes: 60
 
-  #- label: "Run qutip benchmarks with CUDA"
-  #  ...
-  #  env:
-  #    GROUP: "CUDA"
+  ### Python ###
+  - label: "Run QuTiP benchmarks"
+    command: |
+      apt-get update -y
+      apt-get install -y python3-pip
+
+      pip3 install -r benchmark/qutip/requirements.txt
+
+      python3 --version
+      pip3 --version
+
+      python3 -m benchmark.qutip.runbenchmark -v -s
+    agents:
+      queue: "juliagpu"
+      cuda: "*"
+    timeout_in_minutes: 60
```

@Langhaarzombie 

Can you also try if this works ?

```suggestion
  ### Python ###
  - label: "Setup Python"
    key: "python"
    command: |
      apt-get update -y
      apt-get install -y python3-pip
      python3 --version
      pip3 --version

    agents:
      queue: "juliagpu"
      cuda: "*"
    timeout_in_minutes: 60

  - label: "Run QuTiP benchmarks with CUDA"
    depends_on: ["python"]
    command: |
      pip3 install -r benchmark/qutip/requirements.txt
      python3 -m benchmark.qutip.runbenchmark -v -s
    artifact_paths:
      - "benchmark/results/*"
    agents:
      queue: "juliagpu"
      cuda: "*"
    env:
      GROUP: "CUDA"
    timeout_in_minutes: 60
```

---

Tried that but somehow pip3 cannot be found in the next step. That's why I put everything into one.